Server/VikingVault.API/Controllers/AdminController.cs

```diff
@@ -34,6 +35,7 @@ public ActionResult<bool> IsAdmin()
 
         [Route("[action]")]
         [HttpGet]
+        [AuthorizeAdmin]
```

Comment:
Instead of having a filter for  each role, it would be better to have 1 authorization filter that gets as input the role (ex: Authorize("admin")).